litmus-portal/frontend/src/components/CheckBox/index.tsx

```diff
@@ -1,9 +1,29 @@
 import { Checkbox, withStyles } from '@material-ui/core';
 
-const CheckBox = withStyles((theme) => ({
+export const CheckBoxWhite = withStyles((theme) => ({
   root: {
-    color: theme.palette.text.hint,
+    color: theme.palette.border.main,
+    '&$checked': {
+      color: theme.palette.background.paper,
+      '& .MuiSvgIcon-root': {
+        background: theme.palette.secondary.main,
+        borderRadius: '0.25rem',
+      },
+    },
   },
+  checked: {},
```

Comment:
why is this class empty?

Reply:
Without adding this, while overriding the base styles the `&$checked` selector does not work anymore for this MUI component. @arkajyotiMukherjee